x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/signals_alert_type.ts

```diff
@@ -84,11 +102,28 @@ export const signalsAlertType = ({ logger }: { logger: Logger }): AlertType => {
 
         // TODO: Comment this in eventually and use this for manual insertion of the
         // signals instead of the ReIndex() api
-        // const result = await services.callCluster('search', query);
-        const result = await services.callCluster('reindex', reIndex);
 
-        // TODO: Error handling here and writing of any errors that come back from ES by
-        logger.info(`Result of reindex: ${JSON.stringify(result, null, 2)}`);
+        if (process.env.USE_REINDEX_API === 'true') {
+          // eslint-disable-next-line
```

Comment:
You don't need the eslint-disable here.

Reply:
Thanks not sure how that snuck in there 👍